packages/tools/src/eventDispatchers/mouseEventHandlers/mouseMove.ts

```diff
@@ -17,7 +17,7 @@ const { Active, Passive } = ToolModes;
  *
  * @param evt - The normalized mouseDown event.
  */
-export default function mouseMove(evt: MouseMoveEventType) {
+export default function mouseMove(evt: InteractionEventType) {
```

Comment:
Since this is a mouse event, we should use the mouse move event.

Reply:
The dispatchers will still use the mouse event or touch events to be specific. However the handlers will use the general event to be as applicable as needed. We can add this philosophy to the doc too.